frontend/src/Episode/Summary/EpisodeAiring.js

```diff
@@ -39,38 +40,38 @@ function EpisodeAiring(props) {
   if (!showRelativeDates) {
     return (
       <span>
-        {moment(airDateUtc).format(shortDateFormat)} at {time} on {networkLabel}
+        {translate('AirsDateAtTimeOn', { date: moment(airDateUtc).format(shortDateFormat), time })} {networkLabel}
```

Comment:
`networkLabel` should be part of the translation string in case the language prefers `On {networkLabel}, {date} at {time}`

Reply:
networkLabel is an object, not a string so cant be passed through `translate` - unless you want to update `translate`?

Reply:
😢 ahh, similar to our previous conversation on the markdown stuff, we probably do need to do a tag inserter at some point, but for now I think we can cheat a bit and add it like we're going to insert it, but just pass in an empty string, that way the translations will already support it when we improve upon it.

Reply:
done, passed in an empty string and made a TODO note